File: Go/BruteForce/1436.go
package main

import (
	"fmt"
	"strconv"
	"strings"
)

func checkEndNumber(num string) bool {
	if strings.Contains(num, "666") {
		return true
	} else {
		return false
	}
}

func main() {
	//reader := bufio.NewReader(os.Stdin)
	num := 0
	fmt.Scan(&num)
	start := 665
	count := 0
	for count != num {
		start++
		if checkEndNumber(strconv.Itoa(start)) {
			count++
		}
	}
	fmt.Print(start)
}

File: python/BruteForce/1436.py

def checkEndNumber(num:str) ->bool:
    return True if "666" in num else False
    # num 문자가 666d을 포함하고 있으면 return true
if __name__ == "__main__":
    num=int(input())
    start=665
    count=0
    while (count!=num):
        start+=1
        if(checkEndNumber(str(start))):
            count+=1
       
    print(start)